src/main/scala/org/constellation/storage/AddressService.scala

```diff
@@ -5,33 +5,41 @@ import cats.implicits._
 import org.constellation.primitives.Schema.{Address, AddressCacheData}
 import org.constellation.primitives.Transaction
 import org.constellation.primitives.concurrency.MultiLock
-import org.constellation.util.Metrics
 
-class AddressService[F[_]: Concurrent]()(implicit metrics: () => Metrics)
-    extends StorageService[F, AddressCacheData]() {
+class AddressService[F[_]: Concurrent]() {
```

Comment:
Why you've removed the extending by StorageService?

Reply:
In favor of composition.